sp800-63a/sec9_usability.md

```diff
@@ -6,34 +6,36 @@
 
-[ISO/IEC 9241-11](#ISO9241-11) defines usability as: Extent to which a product can be used by specified users to achieve specified goals with effectiveness, efficiency and satisfaction in a specified context of use. This definition focuses on users, goals, and context of use as key elements necessary for achieving effectiveness, efficiency, and satisfaction. A holistic approach considering these key elements is necessary to achieve usability. 
+[ISO/IEC 9241-11](#ISO9241-11) defines usability as the "extent to which a product can be used by specified users to achieve specified goals with effectiveness, efficiency and satisfaction in a specified context of use." This definition focuses on users, goals, and context of use as key elements necessary for achieving effectiveness, efficiency, and satisfaction. A holistic approach considering these key elements is necessary to achieve usability. 
 
-The overarching goal of usability for enrollment and identity proofing is to promote a smooth, positive enrollment process for users by minimizing user burden (e.g. time and frustration) and enrollment friction (e.g. the number of steps to complete and amount of information to track). 
+The overarching goal of usability for enrollment and identity proofing is to promote a smooth, positive enrollment process for users by minimizing user burden (e.g., time and frustration) and enrollment friction (e.g., the number of steps to complete and amount of information to track). 
 
 The enrollment and identity proofing process sets the stage for a user’s interactions with a given authenticator for access to an online service; since negative first impressions can influence user perception of subsequent interactions, organizations benefit from promoting a positive user experience throughout the process. 
 
 An organization’s first step to promoting a smooth, positive enrollment process is familiarizing itself with the user. This consists of matching users with suitable CSPs based on their characteristics ahead of the enrollment and identity proofing process. 
 
+>**MG: this is kind of a federation comment. Here we're only really talking about a single CSP. Worth editing? **
+
 The enrollment and identity proofing process should be designed and implemented so it is easy for users to do the right thing and to recover when something goes wrong, and hard to do the wrong thing or impersonate an individual. Performing a usability evaluation on the enrollment and identity proofing process is critical, conducting it with representative users, realistic goals and tasks, and appropriate contexts of use. 
 
 The current section takes this holistic view in applying usability principles to the enrollment and identity proofing process, with the goal of raising implementers’ awareness of usability considerations associated with enrollment and identity proofing (for usability considerations for typical authenticator usage and intermittent events, see 800-63-3B).
 
 ##### ASSUMPTIONS 
```

Comment:
@garciamike this comment confuses me.  I see this as agnostic to federation.  But I dislike this statement: "The enrollment and identity proofing process sets the stage for a user’s interactions with a given authenticator for access to an online service."